integration/lql_test.go

```diff
@@ -0,0 +1,169 @@
+	}
+	out, err, exitcode := LaceworkCLI("help", "query", "run")
+	assert.Contains(t, out.String(), "lacework query run [query_id] [flags]")
+	assert.Contains(t, out.String(), "--end string")
+	assert.Contains(t, out.String(), "-f, --file string")
+	assert.Contains(t, out.String(), "-r, --repo")
+	assert.Contains(t, out.String(), "--start string")
+	assert.Contains(t, out.String(), "-u, --url string")
+	assert.Contains(t, out.String(), "--validate_only")
+	assert.Empty(t, err.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
+}
+
+func TestQueryRunEditor(t *testing.T) {
+	if env := os.Getenv("CI_BETA"); env == "" {
+		t.Skip("skipping test in production mode")
+	}
+	out, err, exitcode := LaceworkCLIWithTOMLConfig("query", "run")
+	assert.Contains(t, out.String(), "Type a query to run")
+	assert.Contains(t, out.String(), "[Enter to launch editor]")
+	assert.Contains(t, err.String(), "ERROR unable to run LQL query: EOF")
+	assert.Equal(t, 1, exitcode, "EXITCODE is not the expected one")
+}
+
+func TestQueryRunID(t *testing.T) {
+	if env := os.Getenv("CI_BETA"); env == "" {
+		t.Skip("skipping test in production mode")
+	}
+	// setup
+	LaceworkCLIWithTOMLConfig("query", "create", "-u", lqlQueryURL)
+	// teardown
+	defer LaceworkCLIWithTOMLConfig("query", "delete", lqlQueryID)
+
+	// run
+	out, stderr, exitcode := LaceworkCLIWithTOMLConfig(
+		"query", "run", lqlQueryID, "--start", lqlQueryStart, "--end", lqlQueryEnd)
+	assert.Contains(t, out.String(), `"INSERT_ID"`)
+	assert.Empty(t, stderr.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
+}
+
+func TestQueryRunFile(t *testing.T) {
+	if env := os.Getenv("CI_BETA"); env == "" {
+		t.Skip("skipping test in production mode")
+	}
+	// get temp file
+	file, err := ioutil.TempFile("", "TestQueryRunFile")
+	if err != nil {
+		t.FailNow()
+	}
+	defer os.Remove(file.Name())
+
+	// write-to and close file
+	_, err = file.Write([]byte(lqlQueryText))
+	if err != nil {
+		t.FailNow()
+	}
+	file.Close()
+
+	// run
+	out, stderr, exitcode := LaceworkCLIWithTOMLConfig(
+		"query", "run", "-f", file.Name(), "--start", lqlQueryStart, "--end", lqlQueryEnd)
+	assert.Contains(t, out.String(), `"INSERT_ID"`)
+	assert.Empty(t, stderr.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
+}
+
+func TestQueryRunURL(t *testing.T) {
+	if env := os.Getenv("CI_BETA"); env == "" {
+		t.Skip("skipping test in production mode")
+	}
+	// run
+	out, stderr, exitcode := LaceworkCLIWithTOMLConfig(
+		"query", "run", "-u", lqlQueryURL, "--start", lqlQueryStart, "--end", lqlQueryEnd)
+	assert.Contains(t, out.String(), `"INSERT_ID"`)
+	assert.Empty(t, stderr.String(), "STDERR should be empty")
+	assert.Equal(t, 0, exitcode, "EXITCODE is not the expected one")
+}
+
+func TestQueryRunValidateOnly(t *testing.T) {
+	if env := os.Getenv("CI_BETA"); env == "" {
```

Comment:
you should be able to do this instead:
```suggestion
	if os.Getenv("CI_BETA") == "" {
```

Reply:
Man was going whole hog on the inline assignment stuff, but obviously when it only returns a single value and you don't need an assignment this is obv more clear.

Reply:
actually, `os.Getenv` does not return a boolean, which is why I was doing this...

Reply:
Fixed.